tests/TestingTools/ApprovalTestHelpers.ts

```diff
@@ -45,7 +45,7 @@ export function verifyQuery(instructions: string, options?: Options): void {
  *
  * This method explains only the instructions in a single query block.
  *
- * See {@link verifyTaskBlockExplanation} to also explain any global filter or global query.
+ * See {@link verifyTaskBlockExplanation} to also explain the Global Filter or global query.
```

Comment:
The original wording 'any' allowed for there not being a global filter.

I believe that this change changes the meaning of the sentence.

So the description in the PR says it's only capitalisation, but in fact more care is needed in reviewing, to check for changes in meaning too.

Reply:
Shall this take too much of your time, please feel free to close and discard this PR.